cmd/api/main.go

```diff
@@ -211,6 +211,14 @@ func createChainIDStack(
 		eventprocessor.WithHashCalcStep(config.HashCalculationStep),
 	}
 
+	// Add the webhook config if it is enabled for this chain.
+	if config.EventProcessor.Webhook.Enabled {
```

Comment:
If the chain has webhooks enabled, we configure the `EventProcessor` accordingly.